airflow/www/static/js/dag/details/dagRun/ClearRun.tsx

```diff
@@ -59,31 +60,67 @@ const ClearRun = ({ runId, ...otherProps }: Props) => {
     onQueue({ confirmed: true });
   };
 
-  useKeysPress(keyboardShortcutIdentifier.dagRunClear, clearExistingTasks);
+  const [showConfirmationModal, setShowConfirmationModal] = useState(false);
+  const [doNotShowAgain, setDoNotShowAgain] = useState(false);
+
+  useEffect(() => {
+    const storedValue = localStorage.getItem("doNotShowAgain");
```

Comment:
Let's have a more descriptive name for these local storage items to avoid confusion. Maybe like
"doNotShowClearRunModal"